### PR TITLE
Azure VMware Support addendum

### PR DESCRIPTION
**Pull request by varunhari92**
# Pull request guidance

Thank you for your contribution. The content team is monitoring the pull requests. Once your pull request is ready for review, please add a comment such as "PR is ready for review".

### CODE REVIEW

**Comment by suresh-kandoth** [comment]
@HaiyingYu : You can work with @ujpat and Kinye Yu who are involved with these efforts.

File: support/sql/general/support-policy-hardware-virtualization-product.md [hunk 1]
```diff
@@ -172,13 +172,21 @@ The following restrictions and limitations may affect the support policy of the
 
     - SQL Server FCI n Windows Server 2016 and later versions with Azure shared disks. For more information review [Create an FCI with Azure shared disks (SQL Server on Azure VMs)](/azure/azure-sql/virtual-machines/windows/failover-cluster-instance-azure-shared-disks-manually-configure).
 
-- **Q9: Can customers run SQL Server in the Microsoft Azure VM role?**
+- **Q9: Does Microsoft support SQL Server running in Microsoft Azure VMware Solution?**
 
-  A9:  Microsoft Azure VM role is a non-persistent role and not the same as Microsoft Azure Virtual Machine. It is not supported for SQL Server production use. Customers who want to deploy data platform capabilities today in the Microsoft Azure platform should use Microsoft Azure Virtual Machine or Microsoft Azure SQL Database.
+  A9: Yes. Microsoft supports SQL Server 2008 and later versions in Microsoft Azure Infrastructure Services that includes Microsoft Azure VMware Solution. When you deploy SQL Server 2008 and later versions in Microsoft Azure VMware Solution, see:
```
Comment (akashdubey-ms) [inline support/sql/general/support-policy-hardware-virtualization-product.md:177]: Change to: "Yes, Microsoft supports..."

Also, replace: 

- "Services that includes..." with "Services, which includes".
- "....later versions in Microsoft Azure VMware Solution, see:" with "...later versions in Microsoft Azure VMware Solution, consider the following resources:"

File: support/sql/general/support-policy-hardware-virtualization-product.md [hunk 2]
```diff
@@ -172,13 +172,21 @@ The following restrictions and limitations may affect the support policy of the
...
+
+- **Q10: Can customers run SQL Server in the Microsoft Azure VM role?**
+
+  A10:  Microsoft Azure VM role is a non-persistent role and not the same as Microsoft Azure Virtual Machine. It's not supported for SQL Server production use. Customers who want to deploy data platform capabilities today in the Microsoft Azure platform should use Microsoft Azure Virtual Machine or Microsoft Azure SQL Database.
 
-- **Q10: Are there any recommended configuration or best practices to consider while deploying SQL Server in virtualized environments?**
+- **Q11 Are there any recommended configuration or best practices to consider while deploying SQL Server in virtualized environments?**
 
-  A10: Yes, you should consult with the following recommendations from the respective hypervisor:
+  A11: Yes. You should consult with the following recommendations from the respective hypervisor:
```
Comment (akashdubey-ms) [inline support/sql/general/support-policy-hardware-virtualization-product.md:189]: Change to: "Yes, you..."

**Review by akashdubey-ms — COMMENTED** [review]
Looks good

**Comment by akashdubey-ms** [comment]
#sign-off

**Comment by akashdubey-ms** [comment]
@HaiyingYu can you please sign off?

**Comment by sevend2** [comment]
> @HaiyingYu can you please sign off?

Thanks for your review @akashdubey-ms. I will merge this PR once everything is good to go.

**Pull request merged**